scene/animation/tween.cpp

```diff
@@ -1301,8 +1295,7 @@ bool Tween::interpolate_callback(Object *p_object, real_t p_duration, String p_c
 	ERR_FAIL_COND_V(p_duration < 0, false);
 
 	// Check whether the object even has the callback
-	ERR_EXPLAIN("Object has no callback named: %s" + p_callback);
-	ERR_FAIL_COND_V(!p_object->has_method(p_callback), false);
+	ERR_FAIL_COND_V_MSG(!p_object->has_method(p_callback), false, "Object has no callback named: %s" + p_callback + ".");
```

Comment:
These errors were (and still are) bogus. There's no substitution done. This was meant to be passed to `vformat` I guess, but here you can simply drop the `%s` (same below).